### DISCUSSION TITLE
Error - psycopg2.InterfaceError: connection already closed

### DISCUSSION THREAD

**Discussion by Govind10g** [psycopg/psycopg2#1699] [Q&A]
Hi Team,

We are facing **intermittent network connection issue** between our application pod and AWS RDS postgres database.

**Error is application log** - 
[TXN OPERROR] {_get_state_groups_from_groups-13a0a} SSL SYSCALL error: EOF detected
psycopg2.InterfaceError: connection already closed.
**Error is RDS log** - could not receive data from client: Connection reset by peer.

Any idea about this issue ? 
What would be the reason behind it ?

Any insights or assistance you can provide would be greatly appreciated :pray:

**Comment by dvarrazzo** [comment]
We don't offer RDS support for free. If you want a professional consultancy please contact me privately.
